README.md

```diff
@@ -64,3 +67,22 @@ podman run -p 9080:9080 icr.io/appcafe/open-liberty/samples/getting-started
 Open a browser to http://localhost:9080
 
 ![image](https://user-images.githubusercontent.com/3076261/117993383-4f34c980-b305-11eb-94b5-fa7319bc2850.png)
+
+## Run the functional tests
+
+The test cases uses [JUnit 5](https://junit.org/junit5/) and 
+[Maven Failsafe Plugin](https://maven.apache.org/surefire/maven-failsafe-plugin/index.html) defined 
+in [`pom.xml`](pom.xml).
+
+> Note: Sample appplication must be running on `http://localhost` before running the test cases. 
+> <br>
+> See [`HealthUtilIT.java`](src/test/java/it/io/openliberty/sample/health/HealthUtilIT.java) to change 
+> the change the sample application target URL.
+
+To run the test cases against a running sample application, use the following command
+```
+mvnw failsafe:integration-test
```

Comment:
if they're running in dev mode as per the instructions above, the tests should execute just by hitting `Enter` key.

Reply:
Opened https://github.com/OpenLiberty/sample-getting-started/issues/70 to shift to dev mode as test runner